Loop/AppDelegate.swift

```diff
@@ -22,6 +22,7 @@ final class AppDelegate: UIResponder, UIApplicationDelegate, WindowProvider {
 
         loopAppManager.initialize(windowProvider: self, launchOptions: launchOptions)
         loopAppManager.launch()
+        UIApplication.shared.registerForRemoteNotifications()
```

Comment:
This is already happening in NotificationManager; we should probably only do this once.

Reply:
@ps2  This is part of the proposed fix outlined in [1536](https://github.com/LoopKit/Loop/issues/1536) among others). From what I can tell, we _only_ fetch the token via `registerForRemoteNotifications` during onboarding. Since we can't persist the token per iOS guidelines, it will be available in-memory only and not be available for later upload after an app reboot. Here's the cases where that could pose an issue:

1. User chooses to setup Nightscout after onboarding.
2. User goes offline during onboarding process.

I think attempting to fetch it on each startup would be good protection against the above scenarios -- it will always be available to in memory and uploaded to Nightscout when needed.

If that seems reasonable, I agree we don't need it in both places. How about I remove it from onboarding? Or are there other considerations I'm not thinking of?

Reply:
I updated this PR in this way. Please let me know if any any questions on that.